src/odemis/driver/test/zeiss_test.py

```diff
@@ -229,6 +231,122 @@ def test_move(self):
         dur = time.time() - tstart
         self.assertGreaterEqual(dur, 1.1, "1 mm move took only %g s" % dur)
 
+    def test_move_axes6_relative(self):
+        """
+        Test if it's possible to execute a relative move of the stage using 6 axes
+        Test if moving below 0° using a full rotational axis applies correction of underflow
+        Test if moving with an inverted axis works like expected
+        """
+        self.stage.axes["rx"] = model.Axis(unit="rx", range=(0, math.radians(90)))
+        self.stage.axes["rz"] = model.Axis(unit="rz", range=(0, math.radians(360)))
+        self.stage.axes["m"] = model.Axis(unit="m", range=(0, 10.e-3))
```

Comment:
Humm, that's not supposed to be used like that. `.axes` is supposed to be read-only (and it is, if you're using it via the backend). The axes can only be added at init. Please just extend CONFIG_STAGE, as you used to do.

Reply:
Yeah I did actually try to add them to CONFIG_STAGE but then I get problems running the testcases for the 3 axes and 6 axis in the same TestSEM object. In this case the test_move_axes3 is using 6-axes commands for setting and getting due to all the 6 axes registered in self.axes.

Reply:
Do you have any suggestions for a 'smart' way to combine these testcases in one testobject? Or is it just a matter of splitting up 3-axes and 6-axes versions of testcases?

Reply:
Yes, just split up the 2 types. Just create a new TestCase class.